src/gg.regression.unity.experimental.runespawn/Editor/Runtime/SimulationInterface.cs

```diff
@@ -0,0 +1,133 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+using RegressionGames;
+using RegressionGames.StateRecorder;
+using RegressionGames.StateRecorder.Models;
+using RegressionGames.RGLegacyInputUtility;
+
+public static class SimulationInterface
+{
+
+    /// <summary>
+    /// Initializes the simulation interface.
+    /// </summary>
+    /// <param name="monoBehaviour">The MonoBehaviour instance to use for coroutine execution.</param>
+    /// <remarks>
+    /// This method performs the following initializations:
+    /// 1. Starts the RGLegacyInputWrapper simulation.
+    /// 2. Initializes the KeyboardEventSender.
+    /// 3. Initializes the virtual mouse for MouseEventSender.
+    /// </remarks>
+    public static void Initialize(MonoBehaviour monoBehaviour)
```

Comment:
This should also have some sort of teardown method to undo the setup steps from this Initialize method

Reply:
Added a teardown method, mostly mirroring it from BotSegmentsPlaybackController.Stop(). 

Currently, we regain control of mouse and keyboard after calling Teardown.